Source/Tools/FEXServer/Unwind/FileMapping.h

```diff
@@ -1,20 +1,19 @@
 #pragma once
+#include "ELFMapping.h"
+
 #include <cstdint>
+#include <memory>
 #include <string>
 #include <vector>
 
-namespace ELFMapping {
-  struct ELFMemMapping;
-}
-
 namespace FileMapping {
   struct MemMapping;
 
   struct FileMapping {
     uint64_t Begin, End;
     std::string Path;
     std::vector<MemMapping*> MemMappings;
-    ELFMapping::ELFMemMapping *ELFMapping{};
+    std::unique_ptr<ELFMapping::ELFMemMapping> ELFMapping;
```

Comment:
Looks like `std::optional` might be the more fitting abstraction here now, since we don't need heap allocations or ownership transfer.

Reply:
Switched to optional.